src/layouts/event-category-id-layout.js

```diff
@@ -50,15 +52,46 @@ class EventCategoryIdLayout extends React.Component {
     }
   }
 
+  handleParentBreadCrumbs(eventCategory) {
+    const { match } = this.props;
+
+    const breadcrumb = eventCategory.id
+      ? eventCategory.name
+      : T.translate("general.new");
+
+    const baseUrl = match.url.substring(0, match.url.lastIndexOf("/"));
+    const currentBreadcrumbUrl = `${baseUrl}/${eventCategory.id}`;
+
+    if (eventCategory.parent) {
+      const parentUrl = `${baseUrl}/${eventCategory.parent.id}`;
+      return (
+        <>
+          <Breadcrumb
+            data={{
+              title: eventCategory.parent.name,
```

Comment:
if u are only using the parent.name
please check the expand , and relations according